add jb-everything 0.1.0 from pypi

jeromebaum opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io)
or on our [Keybase chat](https://keybase.io/team/condaforge.chat)
if your recipe isn't reviewed promptly.
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found it was in an excellent condition.


I do have some suggestions for making it better though...


For **recipes/jb-everything**:

 * noarch: python recipes are recommended to have a lower bound on the python version. This recommendation will become a requirement in the future.

jeromebaum commented:
I am willing to be listed as maintainer.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found some lint.

Here's what I've got...


For **recipes/jb-everything**:

 * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found some lint.

Here's what I've got...


For **recipes/jb-everything**:

 * Non noarch packages should have python requirement without any version constraints.

synapticarbors commented on recipes/jb-everything/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+  home: https://pypi.org/project/jb-everything/
+  summary: Large list of imports for machine learning.
+  license: MIT
+  license_file: LICENSE  # to be upstreamed
```
I opened a PR (https://github.com/jeromebaum/jb-everything/pull/1) to fix the need to manually include the license file here. Once it is in the source distribution, be sure to remove the local copy. I often recommend putting a comment in the `meta.yaml` as a reminder along the lines of:

```yaml
# License file manually packaged. See https://github.com/jeromebaum/jb-everything/pull/1
license_file: LICENSE
```

as a visual reminder for future updates. Also note that upstream the file is `LICENSE.txt` not `LICENSE`

synapticarbors commented on recipes/jb-everything/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+    - python
+  run:
+    - dabl ~=0.1.9
+    - dask[complete] ~=2.30.0  # to be upstreamed
```
I don't believe we support extras notation. You should just list the extra requirements explicitly.

synapticarbors commented on recipes/jb-everything/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+
+build:
+  number: 0
+  skip: True  # [win]
```
Is there a reason to skip windows and not just use `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

I'm not sure if every dependency is available on windows, but if they are, we should strive to make thie `noarch`.

jeromebaum replied:
Not every dependency is available on Windows - I've added a comment to that effect.

synapticarbors commented on recipes/jb-everything/meta.yaml:
```diff
@@ -0,0 +1,55 @@
+    - missingno ~=0.4.2
+    - numpy ~=1.19.4
+    - pandas ~=1.1.4
+    - python >=3.5  # to be upstreamed
```
If this is not a `noarch: python` recipe, then the restrictions on the python version should be up in the `skip` statement above.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found it was in an excellent condition.


I do have some suggestions for making it better though...


For **recipes/jb-everything**:

 * noarch: python recipes are recommended to have a lower bound on the python version. This recommendation will become a requirement in the future.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found it was in an excellent condition.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found some lint.

Here's what I've got...


For **recipes/jb-everything**:

 * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found some lint.

Here's what I've got...


For **recipes/jb-everything**:

 * Non noarch packages should have python requirement without any version constraints.
 * Non noarch packages should have python requirement without any version constraints.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/jb-everything```) and found it was in an excellent condition.

jeromebaum commented:
@synapticarbors I think I've addressed all of the points (other than the recommendation about the license file comment - shouldn't be long to get that addressed in the upstream package).

jeromebaum commented:
@synapticarbors I also fixed the license comment in line with your suggestion. Do you mind taking another look? Appreciate the holiday season is upon us so no worries if not. Happy holidays! :)
